starboard/evergreen/shared/platform_configuration/BUILD.gn

```diff
@@ -13,20 +13,23 @@
 # limitations under the License.
 
 config("platform_configuration") {
-  ldflags = [
-    "-fuse-ld=lld",
-    "-Wl,--build-id",
-    "-Wl,--gc-sections",
-    "-Wl,-X",
-    "-Wl,-v",
-    "-Wl,-eh-frame-hdr",
-    "-Wl,--fini=__cxa_finalize",
-    "-Wl,-shared",
-    "-Wl,-L$clang_base_path",
-    "-Wl,-L/usr/lib",
-    "-Wl,-L/lib",
-    "-Wl,-u GetEvergreenSabiString",
-  ]
+  ldflags = []
+  if (sb_is_evergreen) {
+    ldflags += [
+      "-fuse-ld=lld",
+      "-Wl,--build-id",
+      "-Wl,--gc-sections",
+      "-Wl,-X",
+      "-Wl,-v",
+      "-Wl,-eh-frame-hdr",
+      "-Wl,--fini=__cxa_finalize",
+      "-Wl,-shared",
```

Comment:
-shared flag will be needed by modular 

ideally should be refactored out in a modular folder in build/config/ so that other platforms can use this and some of the other common compiler flags : can be done in a later cl